### PR TITLE
feat(doc): Updated text selection color

### PR DESCRIPTION
**Pull request by ConradJChan**
Updated text selection background-color to box blue with 10% opacity

Before:
Screen Shot 2020-08-28 at 10 37 14 AM

After:
Screen Shot 2020-08-28 at 11 24 14 AM

### CODE REVIEW

File: src/lib/viewers/doc/DocFindBar.scss [hunk 1]
```diff
@@ -122,19 +121,15 @@ $blue: #00f !default;
     }
 }
 
-.textLayer {
-    opacity: 1;
+.bp-doc {
+    .textLayer {
+        .highlight {
```
Comment (jstoffan) [inline src/lib/viewers/doc/DocFindBar.scss:126]: Does this affect embedded PDF annotations, as well?

File: src/lib/viewers/doc/_docBase.scss [hunk 2]
```diff
@@ -252,14 +252,15 @@ $thumbnail-sidebar-width: 226px;
...
         bottom: auto;
+        opacity: 1;
 
         // Only allow text divs to be selected
         > div {
             user-select: text;
         }
 
-        .highlight {
-            display: inline-block;
```
Comment (jstoffan) [inline src/lib/viewers/doc/_docBase.scss:262]: This seems like kind of a risky change. Do we understand all the use cases for this class?

File: src/lib/viewers/doc/_docBase.scss [hunk 3]
```diff
@@ -252,14 +252,15 @@ $thumbnail-sidebar-width: 226px;
     .textLayer {
         top: 15px; // Match 15px padding top on page
         bottom: auto;
+        opacity: 1;
```
Comment (jstoffan) [inline src/lib/viewers/doc/_docBase.scss:255]: Why is this needed?

File: src/lib/viewers/doc/_docBase.scss (hunk 2 above)
Comment (ConradJChan) [reply src/lib/viewers/doc/_docBase.scss:262]: I just moved it from _docBase.scss to L127 of DocFindBar.scss

File: src/lib/viewers/doc/_docBase.scss (hunk 3 above)
Comment (ConradJChan) [reply src/lib/viewers/doc/_docBase.scss:255]: This is needed because pdfjs styles it with `opacity: .2` so all colors get .2 transparency. But the colors we'll be applying need more control than that

File: src/lib/viewers/doc/_docBase.scss (hunk 2 above)
Comment (jstoffan) [reply src/lib/viewers/doc/_docBase.scss:262]: It looks like we reduced its specificity in the process, though. Given the high usage of these components, can we limit the changes as much as possible to reduce risk?

File: src/lib/viewers/doc/_docBase.scss (hunk 3 above)
Comment (jstoffan) [reply src/lib/viewers/doc/_docBase.scss:255]: I suspect this value is set by pdfjs as a default for a reason, though. Does it have any consequences on the overall user experience? It looks like the test document is mostly text with few images.

File: src/lib/viewers/doc/DocFindBar.scss (hunk 1 above)
Comment (ConradJChan) [reply src/lib/viewers/doc/DocFindBar.scss:126]: Those seem to live on their own layer `annotationLayer`

File: src/lib/viewers/doc/_docBase.scss (hunk 3 above)
Comment (ConradJChan) [reply src/lib/viewers/doc/_docBase.scss:255]: Looking at https://github.com/mozilla/pdfjs-dist/blob/c17c20fb126fcb284811616662156022bbde1947/web/pdf_viewer.css, the only elements of the .textLayer that is affected by this `opacity` was the find bar's highlights which is what I tried to account for below

File: src/lib/viewers/doc/DocFindBar.scss (hunk 1 above)
Comment (ConradJChan) [reply src/lib/viewers/doc/DocFindBar.scss:126]: I played around with a few documents with annotations (drawings & links) and things still seemed to be working well

File: src/lib/viewers/doc/_docBase.scss (hunk 3 above)
Comment (jstoffan) [reply src/lib/viewers/doc/_docBase.scss:255]: Gotcha. So long as the text layer itself doesn't have a background color or something, it should be okay.
